docs/apache-airflow-providers-elasticsearch/logging/index.rst

```diff
@@ -38,7 +38,7 @@ First, to use the handler, ``airflow.cfg`` must be configured as follows:
 
     [elasticsearch]
     host = <host>:<port>
-    log_id_template = {dag_id}-{task_id}-{execution_date}-{try_number}
+    log_id_template = {dag_id}-{task_id}-{data_interval_start}-{try_number}
```

Comment:
The logs should be against the logical/execution date, not the data interval start (as data_interval_start is not guaranteed to be unique.

Reply:
At least conceptually it should be against the logcial date, even if _for now_ these have the same value.

Reply:
Maybe we should use `run_id` instead, that (together with `dag_id` and `task_id`) will forever ever be unique.